modules/seo-tools/jetpack-seo-posts.php

```diff
@@ -60,4 +60,36 @@ public static function get_post_custom_description( $post ) {
 
 		return $custom_description;
 	}
-}
+
+	/**
+	 * Registers the self::DESCRIPTION_META_KEY post_meta for use in the REST API.
+	 */
+	public static function register_post_meta() {
+		if ( ! Jetpack_SEO_Utils::is_enabled_jetpack_seo() ) {
+			return;
+		}
+
+		$args = array(
+			'type' => 'string',
+			'description' => __( 'Custom post description to be used in HTML <meta /> tag.', 'jetpack' ),
+			'single' => true,
+			'default' => '',
+			'show_in_rest' => array(
+				'name' => self::DESCRIPTION_META_KEY
+			),
+		);
+
+		register_meta( 'post', self::DESCRIPTION_META_KEY, $args );
```

Comment:
The Calypso editor currently offers a metabox when editing pages as well. Shouldn't we support pages too?

Reply:
Confirming that the previous PR worked for pages, too. Good catch! Can also be follow up PR I think.

Reply:
Is there a follow-up PR for this already?

Reply:
I don't think anyone worked on that yet.